modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/RepublishOaiPmhWorkflowOperationHandler.java

```diff
@@ -22,46 +22,29 @@
 
```

Comment:
Running the workflow "republish metadata" on an published event results in:

job-operation
UpdateMetadata
job-type
org.opencastproject.publication.oaipmh
stack-trace
java.lang.IllegalArgumentException: Flavors or tags parameter must be set
	at org.opencastproject.publication.oaipmh.OaiPmhPublicationServiceImpl.filterMediaPackage(OaiPmhPublicationServiceImpl.java:574)
	at org.opencastproject.publication.oaipmh.OaiPmhPublicationServiceImpl.updateMetadata(OaiPmhPublicationServiceImpl.java:398)
	at org.opencastproject.publication.oaipmh.OaiPmhPublicationServiceImpl.process(OaiPmhPublicationServiceImpl.java:128)
	at org.opencastproject.job.api.AbstractJobProducer$JobRunner.call(AbstractJobProducer.java:282)
	at org.opencastproject.job.api.AbstractJobProducer$JobRunner.call(AbstractJobProducer.java:241)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

Reply:
Oh: The configuration keys used in the republish-metadata workflow were wrong even before this pull request ;-)

Anyway: As this is simple to do, please fix republish-metadata.xml

Reply:
Ok, done.